cypress/cypress/integration/chat.spec.ts

```diff
@@ -76,6 +77,7 @@ describe("Chat", () => {
     cy.get("[data-cy=chat-thread]").should("exist");
     cy.get("[data-cy=input-field]").type("test");
     cy.get("[data-cy=input-send]").trigger("mouseover").click();
+    cy.get("[data-cy=input-send]").trigger("mouseover").click();
```

Comment:
why are we clicking twice? does cypress have a double click (if we really mean double click)?